### PR TITLE
search suggestion items component

### PR DESCRIPTION
**Pull request by cjohannsen**
based on https://github.com/plentymarkets/plugin-ceres/pull/1944

### All changes meet the following requirements
- ~Changelog entry was added~
- [x] Changes have been tested by the author
- [ ] Changes have been tested by the reviewer
- [x] Plugin can be built

@plentymarkets/ceres-io

### CODE REVIEW

File: resources/views/Widgets/Header/TopBarWidget.twig [hunk 1]
```diff
@@ -44,17 +44,34 @@
                 </div>
             {% endif %}
             {{ Twig.print(Twig.call("LayoutContainer.show", ["Ceres::Header.LeftSide"])) }}
-            <div class="top-bar-items {% if searchStyle == "permanent" %}search-permanent d-flex flex-grow-1 flex-column-reverse flex-md-row{% endif %}">
+            <div class="top-bar-items {% if searchStyle == "permanent" %}search-permanent d-flex flex-grow-1 flex-column-reverse flex-md-row align-items-start{% endif %}">
```
Comment (Tim-M-S) [inline resources/views/Widgets/Header/TopBarWidget.twig]: align-items-start wirklich notwendig?

File: resources/views/Widgets/Header/TopBarWidget.twig [hunk 2]
```diff
@@ -218,7 +236,16 @@
             {% if searchStyle == "onDemand" %}
                 <div id="searchBox" class="cmp cmp-search-box collapse">
                     {{ Twig.if("overrideSearchBar | trim is empty") }}
-                        <item-search :show-item-images="{{ showItemImages | json_encode }}" :forward-to-single-item="{{ forwardToSingleItem | json_encode }}"></item-search>
+                        <item-search>
+                            {% if isPreview or children is defined and children.suggestions | trim is not empty %}
+                                <template #autocomplete-suggestions>
+                                    <div class="autocomplete-suggestions shadow bg-white w-100 overflow-auto{% if isPreview %} position-relative{% endif %}"
+                                            {% if isPreview %} data-builder-child-container="suggestions"{% endif %}>
```
Comment (Tim-M-S) [inline resources/views/Widgets/Header/TopBarWidget.twig]:
```suggestion
                                        {% if isPreview %} data-builder-child-container="suggestions"{% endif %}>
```

File: resources/js/src/app/components/itemList/SearchSuggestionItems.vue [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+        {
+            type: Boolean,
+            default: App.config.search.forwardToSingleItem
+        },
+
+        paddingClasses:
+        {
+            type: String,
+            default: "px-3 py-2"
+        },
```
Comment (Tim-M-S) [inline resources/js/src/app/components/itemList/SearchSuggestionItems.vue:45]:
```suggestion
        },
        
```

**Review by Tim-M-S — CHANGES_REQUESTED** [review]

File: resources/views/Widgets/Header/TopBarWidget.twig (hunk 1 above)
Comment (cjohannsen) [reply resources/views/Widgets/Header/TopBarWidget.twig]: done

**Review by Tim-M-S — APPROVED** [review]

**Pull request merged**